### ISSUE TITLE
Entradas "Resto"

### ISSUE THREAD

**Issue by etchegaray** [olvidio/tramity#46]
Cuando una entrada es decretada a una oficina pero no como oficina principal, esa entrada se ve eligiendo la opción "resto". Sería útil que cuando hay entradas pendientes de resolver o de ver en esa opción se reflejara de alguna manera (al estilo de las bolitas "rojas", que son altamente efectivas para ver lo que está pendiente).

**Comment by Jordi59** [comment]
Sí; interesa. Pienso que bastaría con que, en lugar de contar solo las "entadas" que hay en "propias" sumara también las de "resto". Es lo mismo que hace en la pestaña "Borrador", que cuenta los expedientes que hay en "normal" y "en espera"
